server/src/main/java/io/deephaven/server/console/ConsoleServiceGrpcImpl.java

```diff
@@ -226,7 +229,11 @@ public void executeCommand(
                             diff.setErrorMessage(Throwables.getStackTraceAsString(changes.error));
                             log.error().append("Error running script: ").append(changes.error).endl();
                         }
-                        return diff.setChanges(fieldChanges).build();
+                        diff.setChanges(fieldChanges);
+                        diff.setStartTimestamp(startTimestamp);
+                        diff.setEndTimestamp(endTimestamp);
+
+                        return diff.build();
```

Comment:
nothing at all wrong with this style, but can also be a builder style as before:
```suggestion
                        return diff.setChanges(fieldChanges)
                           .setStartTimestamp(startTimestamp)
                           .setEndTimestamp(endTimestamp)
                           .build();
```

Reply:
Modified back to builder style